db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
I think we are talking about the same thing here - Integrating it in to the seed task is good, but the seed task is only run once (at first deploy) i.e. not at each redeploy, so what I'm trying to say is that you'll need to call populate_languages elsewhere too - somewhere that gets run each deploy otherwise the task won't get executed next time you deploy to prod.